robotcontainer.py

```diff
@@ -161,6 +162,17 @@ def configureButtonBindings(self):
         ModifiableJoystickButton(self.operatorInterface.feedScore).whileTrue(
             FeedIntakeToShooter(self.intake)
         )
+
+        ModifiableJoystickButton(self.operatorInterface.prepShotDynamic).whileTrue(
```

Comment:
Why is this separate from the above command (if the math is done right, a still shot should be calculated the same but just with velocities at 0)

Reply:
that was a mistake

Reply:
Actually still wondering why this is a held state

Reply:
because it takes time to rev everything to the right parts, also consider when moving